interfaces/IDeviceInfo.h

```diff
@@ -25,136 +25,271 @@
 
 namespace Thunder {
 namespace Exchange {
-
+    /* @json 1.0.0 */
     struct EXTERNAL IDeviceInfo : virtual public Core::IUnknown {
         enum { ID = ID_DEVICE_INFO };
 
-        virtual ~IDeviceInfo() override = default;
+        struct FirmwareInfo {
```

Comment:
Just for consistency with the others, can we rename this to Firmware i.s.o. FirmwareInfo?The opthers are also wothout info

Reply:
done